os/drivers/audio/alc1019.c

```diff
@@ -47,8 +47,12 @@
 #define ALC1019_I2S_TIMEOUT_MS 200
 
 /* Default configuration values */
+
+/*
+ * It's good to align the buffer size with i2s DMA page size
```

Comment:
I think you mean to say that the buffer size must match i2s size.
Can you pls change align --> match?